spec/services/versioned_files_service_spec.rb

```diff
@@ -702,31 +680,24 @@
         }.to_json)
       end
 
-      it 'update content files and metadata' do
-        service.delete(version: 3)
+      it 'removes the content files and metadata' do
+        service.delete
         # Deletes content files
-        expect(File.exist?("#{content_path}/3e25960a79dbc69b674cd4ec67a72c62")).to be true
+        expect(File.exist?("#{content_path}/3e25960a79dbc69b674cd4ec67a72c62")).to be false
         expect(File.exist?("#{content_path}/5997de4d5abb55f21f652aa61b8f3aaf")).to be false
 
         # Deletes metadata
-        expect(File.read("#{versions_path}/cocina.1.json")).to eq initial_dro.to_json
-        expect("#{versions_path}/cocina.json").to link_to("#{versions_path}/cocina.1.json")
-        expect(File.read("#{versions_path}/public.1.xml")).to include 'publicObject'
-        expect("#{versions_path}/public.xml").to link_to("#{versions_path}/public.1.xml")
+        expect(File.exist?("#{versions_path}/cocina.2.json")).to be false
         expect(File.exist?("#{versions_path}/cocina.3.json")).to be false
+        expect(File.exist?("#{versions_path}/cocina.3.json")).to be false
```

Comment:
Can we just check that the versions_path no longer exists and not have to do all these checks?